L1Trigger/TrackFindingTracklet/python/Customize_cff.py

```diff
@@ -0,0 +1,25 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
Add comment explaining what this does. IIn particular, I'm unsure what fwConfig() is.